### ISSUE TITLE
Incompatibility with aldeed:template-extension@4.0.0

### ISSUE THREAD

**Issue by cunneen** [JoeyAndres/meteor-ionic#79]
The client gets errors referencing $scope if [aldeed:template-extension](https://github.com/aldeed/meteor-template-extension) is included. To reproduce, just:

`meteor add aldeed:template-extension@4.0.0`

**Comment by JoeyAndres** [comment]
@cunneen Yeah, this is because I am using a forked version of aldeed:template-extension, which I believe is jandres:template-extension. I'll fix this.
